packages/celotool/src/e2e-tests/validator_handshake_tests.ts

```diff
@@ -0,0 +1,263 @@
+import { eqAddress } from '@celo/utils/lib/address'
+import { assert } from 'chai'
+import _ from 'lodash'
+import sleep from 'sleep-promise'
```

Comment:
We don't always import `sleep` consistently, but in the `e2e` tests, `utils.ts` seems to be consensus
```suggestion
import { sleep } from './utils'
```